### ISSUE TITLE
Empty target_site_features.csv file

### ISSUE THREAD

**Issue by acarmas1** [weir12/DeepEdit#7]
Hello,

I've been running the test data set for DeepEdit, when I completed the 1.Reads_extract.sh script. I realized that my target_site_reads.txt is different than the file located here in GitHub (DeepEdit/Getting_Started/target_site_reads.txt)

Here is my code: 
![image](https://user-images.githubusercontent.com/33578163/235546469-a4c0f691-ece4-45aa-98af-82e6ca2bd1ed.png)

Here is my output: 
![image](https://user-images.githubusercontent.com/33578163/235546495-0ce911e7-cffe-4ede-bbe1-11548b34fbd2.png)

And here is the file posted in Github:
![image](https://user-images.githubusercontent.com/33578163/235546581-c61de062-5374-4107-9080-0b58bb7b5c31.png)

Even though, both look different I went ahead and run 2.feature_extract.py by using this code: 
![image](https://user-images.githubusercontent.com/33578163/235546690-c1145541-b2cd-41c5-95aa-e0ee1dbdc38f.png)

And when it finished, the code generated an empty target_site_features.csv file. 

Please, I was wondering if you could help me to troubleshoot it, I want to run DeepEdit first with your testdata set and then try it with my own data to detect A-to-I RNA editing sites in my directRNA sequencing reads. 

I'm looking forward to hearing from you.

Best,
Camila

**Comment by LongxianChen** [comment]
Hi Camila,

I guess this may due to that I compressed the nanopore_reads folder into a nanopore_reads.tar.gz file when I uploaded them on Github. 
I have re-uploaded the related files, you may re-clone the files and try again. 

Best,
Longxian Chen

**Comment by acarmas1** [comment]
Hello,

I re-cloned the DeepEdit directory, and by using this code: 

![image](https://user-images.githubusercontent.com/33578163/235807582-bca717e6-6814-4e8d-9ed2-0709e87950ec.png)

I got the same result, my target_site file still looks the same: 

![image](https://user-images.githubusercontent.com/33578163/235807642-ce33a146-eca0-46d2-9b4c-c0bc1970d215.png)

**Comment by LongxianChen** [comment]
I test the code on another server again and it works well. 
So, you may debug on this following two aspects: 
1. Do you have installed samtools in your server ?
2. Does the nanopore.sam file you used in your code is the one that I provided ?

**Comment by acarmas1** [comment]
Hello,

Thank you, I didn't load samtools before. So now I loaded it and I was able to generate the same target_sites.txt file. 

However, when running the 2.feature_extract.py code. 

I got the same target_site_features.csv, but with this variation. 
![image](https://user-images.githubusercontent.com/33578163/235977603-76cb1f3b-5fbb-4562-b281-3e1ac1dd4c2f.png)
I have 6 columns filled with nan values. I checked, and I'm using python 3.8.3.

I uploaded my outputs in this link: https://1drv.ms/f/s!AokqkR3muxL0jvohT7RWoCTBOV3Ncg?e=w25uhy

**Comment by weir12** [comment]
Hi,

Please take a look at the parameter options for tombo re-squiggle in our other repository:

```

https://github.com/weir12/DENA/blob/2c584c9a22f2903a1c44abe9a734fef5e9d158c8/README.md?plain=1#L116

```
In summary, you need to include the` --include-event-stdev` option in the tombo re-squiggle stage to calculate the standard deviation for each base.

Best regards,
Liang Ou

**Comment by acarmas1** [comment]
It worked. I was able to run your test dataset. 

Thank you so much for your help. 

Now I'm going to try it with my data.

**Comment by acarmas1** [comment]
Hello,

I'm currently trying with my own data. 

I'm performing the basecalling. However, I have a question regarding this step:
guppy_basecaller -i DeepEdit/Getting_Started/nanopore_reads/ -s DeepEdit/Getting_Started/nanopore_reads/ --flowcell FLO-MIN106 --kit SQK-RNA001 --cpu_threads_per_caller 1 **--qscore_filtering** --fast5_out

What version of guppy did you use for your basecalling? Cause when I submitted my script to perform the basecalling I got this error.
![image](https://user-images.githubusercontent.com/33578163/236356360-d42987ab-742e-4748-8ed7-ff6c896fc6eb.png)
It looks like qscore filtering is not longer a parameter for the guppy I have, and when I deleted it from my code, I was able to run it. However, I'm worried that this parameter is essential for further steps.
